youtube_dl/extractor/svt.py

```diff
@@ -141,21 +146,26 @@ class SVTPlayIE(SVTPlayBaseIE):
                     )
                     '''
     _TESTS = [{
-        'url': 'http://www.svtplay.se/video/5996901/flygplan-till-haile-selassie/flygplan-till-haile-selassie-2',
-        'md5': '2b6704fe4a28801e1a098bbf3c5ac611',
+        'url': 'https://www.svtplay.se/video/13906852/en-liten-film-om/en-liten-film-torsk-pa-tallinn-en-liten-film-om-ensamhet',
```

Comment:
This is geo restricted, replace with unrestricted. Also add only matching test with restricted URL.
```suggestion
        'url': 'https://www.svtplay.se/video/26194546/det-har-ar-himlen',
```